fm/models/spotify.py

```diff
@@ -8,11 +8,17 @@
 Models for storing Spotify Track data.
```

Comment:
I don't like idea to tangle name of python models to services.. Now it'll become more misleading when models contains also informations from Echo Nest. I've created a new issue here to fix it later on: https://github.com/thisissoon/FM-API/issues/23